modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -442,30 +442,28 @@
                         "example": "Compensation"
                       },
                       "claimDate": {
-                        "type": [
-                          "string",
-                          "null"
-                        ],
                         "format": "date",
+                        "type": "date",
+                        "nullable": true,
```

Comment:
can this be null? I guess for claims that have hit LH but have not been uploaded to BGS?